components/articles/individual-article/ArticlePage.jsx

```diff
@@ -28,7 +28,7 @@ function ArticlePage(props){
     }
     return (<>
     <article>
-       <ArticleContents title={article.title} author={article.author} image={article.article_img_url} body={article.body} created_at={article.created_at} votes={article.votes}/> 
+       <ArticleContents article_id={articleID} title={article.title} author={article.author} image={article.article_img_url} body={article.body} created_at={article.created_at} votes={article.votes}/> 
```

Comment:
could you simply pass down article={article} and access the properties inside the article contents component, saves a lot of code potentially

Reply:
I don't think so, since article is an object, and React doesn't tend to let you pass objects around as props. I could perhaps pass down the keys and the values separately as two arrays, though?

Reply:
Actually, I'm wrong - that does work.